test/unit/spec/ColorsUtil_spec.js

```diff
@@ -0,0 +1,86 @@
+define([
+  'okta',
+  'helpers/util/Expect',
+  'util/ColorsUtil',
+  'util/Enums'
+],
+function (Okta, Expect, ColorsUtil, Enums) {
+  var $ = Okta.$;
+
+  var normalize = function (text) {
+    return text.replace(/\s+/g, ' ');
+  };
+
+  Expect.describe('ColorsUtil', function () {
```

Comment:
I doubt we need `Expect.describe` but simple `describe` would work best.
(looks like it has already inconsistent usage)